draft-ietf-rats-architecture.md

```diff
@@ -133,35 +133,35 @@ This document uses the following terms.
 
 Appraisal Policy for Evidence:
 
-: A set of rules that informs how a Verifier evaluates the validity of information about an Attester. Compare /security policy/ in {{RFC4949}}
+: A set of rules, reference values and such that is used by a Verifier to evaluate the Evidence from an Attester. Compare /security policy/ in {{RFC4949}}
 
 Appraisal Policy for Attestation Results:
 
 : A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results.
 
 Attester:
 
-: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation
+: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
 
 Claim:
 : A piece of asserted information, often in the form of a name/value pair.
 (Compare /claim/ in {{RFC7519}})
 
 Endorsement:
 
-: A secure statement that some entity (typically a manufacturer) vouches for the integrity of an Attester's signing capability
+: A secure statement that an Endorser (typically a manufacturer) vouches for the integrity of an Attester. It may include implied Claims.
```

Comment:
```suggestion
: A secure statement that an Endorser (typically a manufacturer) vouches for the integrity of an Attester. It may include implied Claims
```